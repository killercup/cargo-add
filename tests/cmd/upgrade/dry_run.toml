bin.name = "cargo-upgrade"
args = ["upgrade", "--dry-run"]
status = "success"
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
