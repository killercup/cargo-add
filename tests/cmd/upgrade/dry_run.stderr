Starting dry run. Changes will not be saved.
    Upgrading docopt v0.8.0 -> v99999.0.0
